examples/10_stormcast_ensemble_example.py

```diff
@@ -0,0 +1,276 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import OrderedDict
+from datetime import datetime
+from math import ceil
+
+import numpy as np
+import torch
+from loguru import logger
+from tqdm import tqdm
+
+from earth2studio.data import DataSource, fetch_data
+from earth2studio.io import IOBackend
+from earth2studio.models.dx import DiagnosticModel
+from earth2studio.models.px import PrognosticModel
+from earth2studio.perturbation import Perturbation
+from earth2studio.utils.coords import CoordSystem, map_coords, split_coords
+from earth2studio.utils.time import to_time_array
+
+logger.remove()
+logger.add(lambda msg: tqdm.write(msg, end=""), colorize=True)
+
+
+# sphinx - deterministic start
+# %%
+"""
+Running StormCast Ensemble Inference
+===============================
+
+Ensemble StormCast inference workflow.
+
+This example will demonstrate how to run a simple inference workflow to generate a
+ensemble forecast using StormCast. For details about the stormcast model, 
+see 
+
+ - https://arxiv.org/abs/2408.10958
+
+"""
+
+# %%
+# Set Up
+# ------
+# All workflows inside Earth2Studio require constructed components to be
+# handed to them. In this example, let's take a look at the most basic:
+# :py:meth:`earth2studio.run.deterministic`.
```

Comment:
Ref ensemble here instead of deterministic?

Reply:
Fixed, thank you.